README.md

```diff
@@ -31,12 +31,12 @@ brew(gui(App()))
 
 The code above will result in the following application:
 
-<img alt="Counter Application" src="docs/assets/introduction.png" width="228px">
+<img alt="Counter Application" src="../docs/assets/introduction.png" width="228px">
 
 Owlkettle also supports building libadwaita apps.
 To enable libadwaita, import `owlkettle/adw` and change the last line to `adw.brew(gui(App()))`.
 
-<img alt="Counter Application using Adwaita Stylesheet" src="docs/assets/introduction_adwaita.png" width="322px">
+<img alt="Counter Application using Adwaita Stylesheet" src="../docs/assets/introduction_adwaita.png" width="322px">
```

Comment:
This file still needs to work as the README of the GitHub Repository, currently the images are not displayed correctly there.

Reply:
Version with fixed links is now deployed for viewing, I'll mark this as resolved